pkg/api/test/error/error.go

```diff
@@ -0,0 +1,18 @@
+package error
+
+import "testing"
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// AssertErrorMessage asserts that err.Error() is equal to wantMsg.
+func AssertErrorMessage(t *testing.T, err error, wantMsg string) {
```

Comment:
Why duplicate this and the `CloudError` tests? Couldn't we drop the corresponding `/test/.../` files in favour of just these?

Reply:
I'm of two minds about this, honestly... perhaps it is just better to have everything use this.